Hmwk_#3_Hangman/hangman_game.py

```diff
@@ -0,0 +1,159 @@
+import functools
+import random
+from functools import reduce
+import time
+import hangman_graphics
+
+#get data from file but not read so later it can be accesed twice without reopening
+def get_words(filename):
```

Comment:
if you want to add some description to the function, it'd be better to use docstring.
```
def get_words(filename):
    '''get data from file but not read so later it can be accesed twice without reopening'''
```
Then you'll be able to write if you want to know what function does. 

`print(get_words.__doc__)`
More information here: https://proglib.io/p/python-docs (docstring section)